Pokedex_ios/Helpers/GraphQLHelpers/GraphQLResultConverter.swift

```diff
@@ -16,7 +16,9 @@ class ConversionHelper {
             let jsonData = try JSONSerialization.data(withJSONObject: data.jsonObject)
             var decode: DataClass = try JSONDecoder().decode(DataClass.self, from: jsonData)
             for index in 0..<decode.species.count {
+                let pokemonDetails = decode.species[index].pokemonDetails[0]
```

Comment:
Since this is part of my code, I'll answer this. The information retrieved from Apollo is by species and a specie can have more than one pokemon
e.g: deoxys. this specie has 4 variations including the base pokemon